webanno-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/ProjectUsersPanel.java

```diff
@@ -377,6 +404,7 @@ public UserDetailForm(String id)
                 @Override
                 public void onSubmit()
                 {
+                	filterText.setModel(Model.of(""));
```

Comment:
I cannot see the new code. Do you fetch the model from `UserDetailForm`, i.e. `UserDetailForm.this.getModelObject().userFilter = ""`?